### PR TITLE
Update PR pipeline with ciliume2e

### PR DESCRIPTION
**Pull request by camrynl**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

**Comment by camrynl** [comment]
splitting swfte2e and ciliume2e into separate templates in the pipeline for future kube-proxy/non kube-proxy cluster scenarios

File: test/integration/manifests/cns/daemonset.yaml [hunk 1]
```diff
@@ -119,4 +133,4 @@ spec:
         - name: cns-config
           configMap:
             name: cns-config
-      serviceAccountName: azure-cns
+      serviceAccountName: azure-cns
```
Comment (rbtr) [inline test/integration/manifests/cns/daemonset.yaml:136]: you should configure your editor to add trailing newlines on save

File: cilium/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        {
+          "type": "cilium-cni",
+          "ipam": {
+            "type": "azure-ipam"
+          },
+          "enable-debug": true,
+          "log-file": "/var/log/cilium-cni.log"
+        }
+      ],
+      "ProgramSNATIPTables" : true
```
[the remark this answers is not in the document]
Comment (camrynl) [reply cilium/configmap.yaml]: hm I can see this new comment now, not sure why I couldn't see the original one

[the remark this answers is not in the document]
Comment (nddq) [reply cilium/configmap.yaml]: yeah my bad, I did not hit submit review 🥲

**Review by rbtr — APPROVED** [review]

**Pull request merged**